bigtable/integration_test.go

```diff
@@ -2278,92 +2309,118 @@ func TestIntegration_AdminBackup(t *testing.T) {
 		return infos, err
 	}
 
-	if err := adminClient.CreateTable(ctx, table); err != nil {
-		t.Fatalf("Creating table: %v", err)
-	}
-
 	// Precondition: no backups
-	backups, err := list(cluster)
+	backups, err := list(sourceCluster)
 	if err != nil {
-		t.Fatalf("Initial backup list: %v", err)
+		t.Errorf("Initial backup list: %v", err)
 	}
 	if got, want := len(backups), 0; got != want {
-		t.Fatalf("Initial backup list len: %d, want: %d", got, want)
+		t.Errorf("Initial backup list len: %d, want: %d", got, want)
 	}
 
 	// Create backup
 	backupName := "mybackup"
-	defer adminClient.DeleteBackup(ctx, cluster, "mybackup")
 
-	if err = adminClient.CreateBackup(ctx, table, cluster, backupName, time.Now().Add(8*time.Hour)); err != nil {
-		t.Fatalf("Creating backup: %v", err)
+	if err = adminClient.CreateBackup(ctx, tblConf.TableID, sourceCluster, backupName, time.Now().Add(8*time.Hour)); err != nil {
+		t.Errorf("Creating backup: %v", err)
 	}
 
 	// List backup
-	backups, err = list(cluster)
+	backups, err = list(sourceCluster)
 	if err != nil {
-		t.Fatalf("Listing backups: %v", err)
+		t.Errorf("Listing backups: %v", err)
```

Comment:
At least some of these should stay as `Fatalf`-- for example, if there is a non-nil error here, or if there aren't at least 1 backup returned, then continuing with the test will cause a panic. Only use `Errorf` if it's useful for the test to continue execution.

Reply:
Looks like this is still an issue (e.g. listing backups doesn't make sense if `CreateBackup` fails). Might need a push?